examples/QmlBarcodeReader/qml/GeneratorPage.qml

```diff
@@ -0,0 +1,254 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtMultimedia 5.12
+import QtQuick.Layouts 1.12
+import com.scythestudio.sbarcodereader 1.0
+
+Item {
+    id: root
+    anchors.fill: parent
+
+    SBarcodeGenerator {
+        id: barcodeGenerator
+        onProcessFinished: {
+            console.log(barcodeGenerator.filePath);
+            image.source = "file:///" + barcodeGenerator.filePath
+        }
+    }
+
+    Rectangle {
+        anchors.fill: parent
+        height: parent.height
+        width: parent.width
+        visible: true
+        color: "white"
+
+        Rectangle {
+            id: inputRect
+            z: 100
+            anchors.top: parent.top
+            anchors.left: parent.left
+            anchors.right: parent.right
+            height: 40
+            TextField {
+                id: textField
+                placeholderText: qsTr("Input")
+                anchors.fill: parent
+                selectByMouse: true
+                MouseArea {
+                    anchors.fill: parent
+                    cursorShape: Qt.IBeamCursor
+                    acceptedButtons: Qt.NoButton
+                }
+            }
+        }
+
+        Rectangle {
+            id: imageRect
+            anchors.top: inputRect.bottom
+            anchors.bottom: buttonsRect.top
+            anchors.left: parent.left
+            anchors.right: parent.right
+            Image {
+                id: image
+                anchors.centerIn: parent
+                width: parent.width
+                height: image.width
+                cache: false
+            }
+        }
+
+        Rectangle {
+            id: buttonsRect
+            anchors.bottom: parent.bottom
+            anchors.left: parent.left
+            anchors.right: parent.right
+            height: 40
+            RowLayout {
+                id: buttonsLayout
+                anchors.fill: parent
+                Button {
+                    id: settingsButton
+                    anchors.margins: 5
+                    Layout.alignment: Qt.AlignHCenter
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                    visible: true
+                    checkable: true
```

Comment:
I think that these two properties have such values be default -> it's not needed to set them.